test-suite/Makefile

```diff
@@ -118,6 +118,10 @@ clean:
 	$(HIDE)find . \( \
 	  -name '*.stamp' -o -name '*.vo' -o -name '*.vio' -o -name '*.log' \
 	\) -print0 | xargs -0 rm -f
+	$(SHOW) "RM        <**/*.cmo> <**/*.cmi> <**/*.test> <**/*.cache>"
+	$(HIDE)find unit-tests \( \
+	  -name '*.stamp' -o -name '*.cmo' -o -name '*.cmi' -o -name '*.test' -o -name '*.cache' \
+	\) -print0 | xargs -0 rm -f
```

Comment:
Why not use -exec?

Reply:
Good idea. I was copying the existing code.